docs/advanced/settings.md

```diff
@@ -60,6 +60,22 @@ The following example will add disks matching the glob pattern `/dev/sd*` or `/d
 /dev/sd*,/dev/hd*
 ```
 
+## `auto-rotate-rke2-certs`
```

Comment:
@FrankYang0529 Sorry, I think we can also add the starting version:

```
Versions: v1.3.0 and later
```

Reply:
Update it. Thanks.